lucetc/Cargo.toml

```diff
@@ -9,22 +9,19 @@ categories = ["wasm"]
 authors = ["Lucet team <lucet@fastly.com>"]
 edition = "2018"
 
-[lib]
-crate-type=["rlib"]
-
 [[bin]]
 name = "lucetc"
-path = "src/main.rs"
+path = "lucetc/main.rs"
 
 [dependencies]
 bincode = "1.1.4"
-cranelift-codegen = { path = "../cranelift/cranelift-codegen", version = "0.44.0" }
-cranelift-entity = { path = "../cranelift/cranelift-entity", version = "0.44.0" }
-cranelift-native = { path = "../cranelift/cranelift-native", version = "0.44.0" }
-cranelift-frontend = { path = "../cranelift/cranelift-frontend", version = "0.44.0" }
-cranelift-module = { path = "../cranelift/cranelift-module", version = "0.44.0" }
-cranelift-faerie = { path = "../cranelift/cranelift-faerie", version = "0.44.0" }
-cranelift-wasm = { path = "../cranelift/cranelift-wasm", version = "0.44.0" }
+cranelift-codegen = { path = "../cranelift/cranelift-codegen", version = "0.46.1" }
```

Comment:
I ended up not actually changing cranelift to make this work, so this upgrade is just a nice little bonus :sparkles: